### ISSUE TITLE
Docutils cannot render a document where a markdown table uses inline HTML tags.

### ISSUE THREAD

**Issue by RojerGS** [executablebooks/MyST-Parser#315]
Docutils cannot render a document where a markdown table uses inline HTML tags.

**To Reproduce**

I stumbled upon this bug when trying to build a book with the `jupyter-book` tool, so the easiest way to go about this would be to create a sample `jupyter-book` book:

```
python -m pip install jupyter-book
jupyter-book create book
```

Open `book/content.md` and append the following markdown to the end of the `.md` file:

```
| Table |
| :- |
| <i>Italics</i> |
```

then run

```
jupyter-book build book
```

And then an error pops up, that I was able to trace back to the fact that `<i>` is being used inside the table (full traceback below):
```
...
  File "c:\users\rodri\anaconda3\lib\site-packages\myst_parser\sphinx_parser.py", line 195, in parse
    parser.renderer.render(tokens, parser.options, env)
  File "c:\users\rodri\anaconda3\lib\site-packages\myst_parser\docutils_renderer.py", line 154, in render
    self.rules[f"render_{nest_token.type}"](nest_token)
  File "c:\users\rodri\anaconda3\lib\site-packages\myst_parser\docutils_renderer.py", line 715, in render_table_open
    self.render_table_row(cast(Token, body_row))
  File "c:\users\rodri\anaconda3\lib\site-packages\myst_parser\docutils_renderer.py", line 728, in render_table_row
    self.render_children(child)
...
  File "c:\users\rodri\anaconda3\lib\site-packages\myst_parser\docutils_renderer.py", line 527, in render_html_inline
    self.render_html_block(token)
  File "c:\users\rodri\anaconda3\lib\site-packages\myst_parser\docutils_renderer.py", line 530, in render_html_block
    node_list = html_to_nodes(token.content, token_line(token), self)
  File "c:\users\rodri\anaconda3\lib\site-packages\myst_parser\docutils_renderer.py", line 81, in token_line
    raise ValueError(f"token map not set: {token}")
ValueError: token map not set: Token(type='html_inline', tag='', nesting=0, attrs=None, map=None, level=0, children=None, content='<i>', markup='', info='', meta={}, block=False, hidden=False)
```

This happens with several HTML tags, namely `<b>`, `<i>`, `<code>` and `<kbd>`.

<details>
<summary>Full error traceback.</summary>

```
[31;01mException occurred:[39;49;00m
  File "c:\users\rodri\anaconda3\lib\site-packages\myst_parser\docutils_renderer.py", line 81, in token_line
    raise ValueError(f"token map not set: {token}")
ValueError: token map not set: Token(type='html_inline', tag='', nesting=0, attrs=None, map=None, level=0, children=None, content='<i>', markup='', info='', meta={}, block=False, hidden=False)
[31;01mThe full traceback has been saved in C:\Users\rodri\AppData\Local\Temp\sphinx-err-2n92u175.log, if you want to report the issue to the developers.[39;49;00m
Please also report this if it was a user error, so that a better error message can be provided next time.
A bug report can be filed in the tracker at <https://github.com/sphinx-doc/sphinx/issues>. Thanks!
Traceback (most recent call last):
  File "c:\users\rodri\anaconda3\lib\site-packages\jupyter_book\sphinx.py", line 150, in build_sphinx
    app.build(force_all, filenames)
  File "c:\users\rodri\anaconda3\lib\site-packages\sphinx\application.py", line 349, in build
    self.builder.build_update()
  File "c:\users\rodri\anaconda3\lib\site-packages\sphinx\builders\__init__.py", line 299, in build_update
    len(to_build))
  File "c:\users\rodri\anaconda3\lib\site-packages\sphinx\builders\__init__.py", line 311, in build
    updated_docnames = set(self.read())
  File "c:\users\rodri\anaconda3\lib\site-packages\sphinx\builders\__init__.py", line 418, in read
    self._read_serial(docnames)
  File "c:\users\rodri\anaconda3\lib\site-packages\sphinx\builders\__init__.py", line 439, in _read_serial
    self.read_doc(docname)
  File "c:\users\rodri\anaconda3\lib\site-packages\sphinx\builders\__init__.py", line 479, in read_doc
    doctree = read_doc(self.app, self.env, self.env.doc2path(docname))
  File "c:\users\rodri\anaconda3\lib\site-packages\sphinx\io.py", line 316, in read_doc
    pub.publish()
  File "c:\users\rodri\anaconda3\lib\site-packages\docutils\core.py", line 218, in publish
    self.settings)
  File "c:\users\rodri\anaconda3\lib\site-packages\sphinx\io.py", line 130, in read
    self.parse()
  File "c:\users\rodri\anaconda3\lib\site-packages\docutils\readers\__init__.py", line 77, in parse
    self.parser.parse(self.input, document)
  File "c:\users\rodri\anaconda3\lib\site-packages\myst_nb\parser.py", line 59, in parse
    super().parse(inputstring, document=document)
  File "c:\users\rodri\anaconda3\lib\site-packages\myst_parser\sphinx_parser.py", line 195, in parse
    parser.renderer.render(tokens, parser.options, env)
  File "c:\users\rodri\anaconda3\lib\site-packages\myst_parser\docutils_renderer.py", line 154, in render
    self.rules[f"render_{nest_token.type}"](nest_token)
  File "c:\users\rodri\anaconda3\lib\site-packages\myst_parser\docutils_renderer.py", line 715, in render_table_open
    self.render_table_row(cast(Token, body_row))
  File "c:\users\rodri\anaconda3\lib\site-packages\myst_parser\docutils_renderer.py", line 728, in render_table_row
    self.render_children(child)
  File "c:\users\rodri\anaconda3\lib\site-packages\myst_parser\docutils_renderer.py", line 260, in render_children
    self.rules[f"render_{child.type}"](child)
  File "c:\users\rodri\anaconda3\lib\site-packages\myst_parser\docutils_renderer.py", line 334, in render_inline
    self.render_children(token)
  File "c:\users\rodri\anaconda3\lib\site-packages\myst_parser\docutils_renderer.py", line 260, in render_children
    self.rules[f"render_{child.type}"](child)
  File "c:\users\rodri\anaconda3\lib\site-packages\myst_parser\docutils_renderer.py", line 527, in render_html_inline
    self.render_html_block(token)
  File "c:\users\rodri\anaconda3\lib\site-packages\myst_parser\docutils_renderer.py", line 530, in render_html_block
    node_list = html_to_nodes(token.content, token_line(token), self)
  File "c:\users\rodri\anaconda3\lib\site-packages\myst_parser\docutils_renderer.py", line 81, in token_line
    raise ValueError(f"token map not set: {token}")
ValueError: token map not set: Token(type='html_inline', tag='', nesting=0, attrs=None, map=None, level=0, children=None, content='<i>', markup='', info='', meta={}, block=False, hidden=False)

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "c:\users\rodri\anaconda3\lib\runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "c:\users\rodri\anaconda3\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "C:\Users\rodri\anaconda3\Scripts\jb.exe\__main__.py", line 7, in <module>
  File "c:\users\rodri\anaconda3\lib\site-packages\click\core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "c:\users\rodri\anaconda3\lib\site-packages\click\core.py", line 782, in main
    rv = self.invoke(ctx)
  File "c:\users\rodri\anaconda3\lib\site-packages\click\core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "c:\users\rodri\anaconda3\lib\site-packages\click\core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "c:\users\rodri\anaconda3\lib\site-packages\click\core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "c:\users\rodri\anaconda3\lib\site-packages\jupyter_book\commands\__init__.py", line 305, in build
    result, builder, OUTPUT_PATH, build_type, PAGE_NAME, click.echo
  File "c:\users\rodri\anaconda3\lib\site-packages\jupyter_book\commands\__init__.py", line 546, in builder_specific_actions
    raise RuntimeError(_message_box(msg, color="red", doprint=False)) from result
RuntimeError:
[91m===============================================================================[0m

There was an error in building your book. Look above for the cause.

[91m===============================================================================[0m
```

</details>

**Expected behavior**

The book should be built successfully.

**Environment**

- Python Version: 3.7.6
- Package versions or output of `jupyter-book --version`: 0.10.0
- Operating System: Windows 10 64-bit

**Comment by chrisjsewell** [comment]
Hey @RojerGS this was already fixed in v0.13.5 (https://github.com/executablebooks/MyST-Parser/blob/master/CHANGELOG.md#0135---2021-02-15) although feel free to re-open if not

**Comment by RojerGS** [comment]
Slightly late for the party, then! Thanks for the help.
